### ISSUE TITLE
How to stop scrolling when cursor is at the bottom of the screen and the console shrinks

### ISSUE THREAD

**Issue by x1bzzr** [microsoft/terminal#4180]
I'm running into a situation where, when the the cursor is positioned at the bottom of the screen and the user decreases the size of the console window by resizing it, the console will automatically scroll down to keep the cursor within the screen which makes the contents of the screen change. The application will then notice than the console has resized and will move the cursor to the first row in order to update the screen for the new console size, but because previously the screen was scrolled, moving the cursor to the top row will trigger another scroll which again will update the contents of screen. After this the application updates the screen to reflect the new console size. The unintended effect of all this is that the screen is updated three times instead of only once after the console is resized. Is there a way I can avoid this?

**Comment by DHowett-MSFT** [comment]
Hey there! Would you mind filling out our bug template and telling us your OS version? It sounds like you're reporting an issue with the traditional console (and not the new Terminal), so this is pretty important info :smile:

Thanks.

**Comment by x1bzzr** [comment]
OS version is Windows 10, 19013.1102 build. About filling a bug template: I don't know what that means. Sorry, I'm just not very familiar with Github. What is it that I should do exactly?

**Comment by x1bzzr** [comment]
Never mind, I figured it out. The new issue is #4181. Should I delete this one?
